Common.Configurations.targets

```diff
@@ -30,7 +30,7 @@
 	<PackageReference Include="DigitalOffice.Kernel.BrokerSupport" Version="1.0.0" />
 	<PackageReference Include="DigitalOffice.Kernel.ImageSupport" Version="1.0.3" />
 	<PackageReference Include="DigitalOffice.Kernel.EFSupport" Version="1.0.0" />
-	<PackageReference Include="DigitalOffice.Models" Version="1.0.7" />
+	<PackageReference Include="DigitalOffice.Models" Version="1.0.9" />
```

Comment:
1.0.10

Reply:
Models versions grow so fast, so it can be changed with every new PR XD

Reply:
Done